CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v0.120.4, 1 October 2020
+
+- Go: Bump golang to v1.15.2 (thanks, @chenrui333!)
```

Comment:
```suggestion
- Go: Bump golang to v1.15.2
```
AFAICT this was _jurre in https://github.com/dependabot/dependabot-core/pull/2570

Reply:
Thanks for catching that. Total copypasta on my part. 🤦